src/components/modeler/Modeler.vue

```diff
@@ -250,6 +251,13 @@ export default {
     },
   },
   methods: {
+    defaultFlow(flow) {
```

Comment:
Could you rename this to `toggleDefaultFlow` or similar? 

Or at least extract `source.default && source.default.id === flow.id` into a labeled expression that communicate the toggling intent :)

Reply:
Renamed to "toggleDefaultFlow"